src/zenml/logger.py

```diff
@@ -119,6 +118,11 @@ def get_console_handler() -> Any:
     console_handler = logging.StreamHandler(sys.stdout)
     console_handler.setFormatter(CustomFormatter())
     return console_handler
+    # console_handler = RichHandler(
+    #     show_path=False, omit_repeated_times=False, console=console
+    # )
+    # console_handler.setFormatter(CustomFormatter())
+    # return console_handler
```

Comment:
This seems to be forgotten code

Reply:
Actually not. I wanted to leave it in in case we decide to tackle the logging part in the future. Maybe that's not such a great practice, though. Better to just delete?